### PR TITLE
Document the extension engine and exisitng extensions

### PR DESCRIPTION
**Pull request by oleg-nenashev**
Just fixes the current state of extensions in the documentation

@reviewbybees

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ API stability is not guaranteed till the first release, the project structure is
 Major changes since 1.x:
 * Rework of the project structure
 * Better logging
```
Comment (oleg-nenashev) [inline README.md]: TODO: reference the entry in the ToC above

Comment (oleg-nenashev) [inline README.md]: done

**Comment by aheritier** [comment]
🐝

**Comment by oleg-nenashev** [comment]
@reviewbybees done

**Pull request merged**